_data/fa-owners.yml

```diff
@@ -0,0 +1,353 @@
+'API definition':
+  - area: 'API definition'
+    label: api-definition
+    owner:
+    - sq-lb-epitome
+    repos:
+    - loopback-api-definition
+    - loopback-component-explorer
+    - loopback-swagger
+
+'Boot':
+  - area: Boot
+    label: boot
+    owner:
+    - sq-lb-asteroid
+    repos:
+    - loopback-boot
+    - loopback-example-app-logic
+    smes:
+    - davidcheung
```

Comment:
It's trivial to add a `@` but the text is already linked.  Adding `@` serves no real purpose.